### PR TITLE
iOS Deployment target

### LINKED ISSUE #52: Question: Deployment target
Hi. I just discovered this repo and would like to participate.
I would like to know if there is a reason that the the deployment target is iOS 8.0?

I think the codebase could benefit from even little things like StackViews.
Regards

### PR DESCRIPTION
**Pull request by mltbnz**
raises deployment target to 10.0

closes #52

### CODE REVIEW

**Comment by lennet** [comment]
There are a lot of #available in the codebase that are not longer needed. We should remove them

File: CriticalMass/Colors.swift [hunk 1]
```diff
@@ -9,11 +9,7 @@ import UIKit
 
 extension UIColor {
     private class func safeInit(red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat) -> UIColor {
-        if #available(iOS 10.0, *) {
-            return UIColor(displayP3Red: red / 255, green: green / 255, blue: blue / 255, alpha: alpha)
-        } else {
-            return UIColor(red: red / 255, green: green / 255, blue: blue / 255, alpha: alpha)
-        }
+        return UIColor(displayP3Red: red / 255, green: green / 255, blue: blue / 255, alpha: alpha)
```
Comment (lennet) [inline CriticalMass/Colors.swift]: Minor: this helper is no longer needed

Comment (mltbnz) [reply CriticalMass/Colors.swift]: True. Altough I liked the shorter init I removed it to not add another convenience initializer and using one that also Zeplin would produce

**Comment by lennet** [comment]
Is this PR ready to merge from your side?

**Comment by mltbnz** [comment]
It is. Silently squeezed two updates in which where fitting in this one 🤫

**Comment by lennet** [comment]
I found a small issue, which can cause a warning. The Deployment-target in the pod file still uses 8.0

**Comment by mltbnz** [comment]
Merged it. Is there a policy btw.? Do I merge when you approve?

**Comment by headione** [comment]
> Merged it. Is there a policy btw.? Do I merge when you approve?

you can merge after 1 thumb @mltbnz

**Pull request merged**